### PR TITLE
Feature/transfer funds functionality

### LINKED ISSUE #82: transfer funds functionality

### PR DESCRIPTION
**Pull request by obasilakis**
Closes #82

### CODE REVIEW

File: src/shared/constants/nodeData.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    url: 'wss://rpc.cyprus1.colosseum.quaiscan.io',
+    provider: 'https://rpc.cyprus1.colosseum.quaiscan.io',
+  },
+  'wallet-zone-0-1': {
+    url: 'wss://rpc.cyprus2.colosseum.quaiscan.io',
+    provider: 'https://rpc.cyprus2.colosseum.quaiscan.io',
+  },
+  'wallet-zone-0-2': {
+    url: 'wss://rpc.cyprus3.colosseum.quaiscan.',
+    provider: 'https://rpc.cyprus2.colosseum.quaiscan.io',
```
Comment (juanmanso) [inline src/shared/constants/nodeData.ts:37]: question: shouldn't this be with domain `cyprus3`?

```suggestion
    provider: 'https://rpc.cyprus3.colosseum.quaiscan.io',
```

File: src/shared/services/transferFunds.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+export const transferFunds = async (to: string, amount: string) => {
+  const zone = getZone();
+  const sendNodeData = allNodeData[zone];
+  const walletData = await retrieveWallet();
+
+  const provider = new quais.providers.JsonRpcProvider(sendNodeData.provider);
+  await provider.ready;
+
+  const valueToTransferInWei = quais.utils.parseEther(amount);
```
Comment (juanmanso) [inline src/shared/services/transferFunds.ts:13]: nitpick: is this a typo on `Wei` (rather than `Gwei`)?

File: src/shared/hooks/useWallet.ts [hunk 3]
```diff
@@ -1,8 +1,9 @@
 import { useEffect, useState } from 'react';
 import { retrieveWallet } from '../services/retrieveWallet';
+import { Wallet } from '../types/Wallet';
 
 export const useWallet = () => {
-  const [wallet, setWallet] = useState<any>();
+  const [wallet, setWallet] = useState<Wallet>();
```
Comment (juanmanso) [inline src/shared/hooks/useWallet.ts:6]: suggestion: could we initialise the wallet with some default blank value? By doing so, we avoid breaking the app if we want to access `wallet.address` and it hasn't been retrieved yet. If not, we could still check for it to exist but IMO it's something we could avoid if we want to

**Review by juanmanso — COMMENTED** [review]
Left an important question (at least for me) on `nodeData.ts` but the other ones are suggestions or nitpicks that do not block the Pr

File: src/shared/services/transferFunds.ts (hunk 2 above)
Comment (obasilakis) [reply src/shared/services/transferFunds.ts:13]: nah, parseEther returns the value in Wei

File: src/shared/hooks/useWallet.ts (hunk 3 above)
Comment (obasilakis) [reply src/shared/hooks/useWallet.ts:6]: fair point, will change

File: src/shared/constants/nodeData.ts (hunk 1 above)
Comment (obasilakis) [reply src/shared/constants/nodeData.ts:37]: good catch!

**Review by hsyndeniz — APPROVED** [review]

**Pull request merged**